src/script/cpp_api/s_base.h

```diff
@@ -124,6 +124,7 @@ class ScriptApiBase : protected LuaHelper {
 	friend class ModApiBase;
 	friend class ModApiEnvMod;
 	friend class LuaVoxelManip;
+	friend class TestMoveAction; // needs getStack()
```

Comment:
Maybe add `check_function` to `cpp_api` instead? Making all tests friend classes looks bad to me.

Reply:
Yes, it is a bit hacky, however I would prefer to not integrate unittest code into the regular code base. The current approach reduces code complexity and hence maintenance. Furthermore, `check_function` is pretty trivial but might change for other unittests (different amount of parameters or return values).